docs/implementation-guide.md

```diff
@@ -171,7 +153,7 @@ To create a repository secret follow this steps.
 5. Enter a name for your development environment (e.g. `dev`) and click `Configure environment`. No additional configuration of the environment is required at this time you proceed to step 9.
 ![create-environment-3](./images/create-environment-3.png)
 
-### Step 9: Run the Deployment GitHub Workflow
+### Step 6: Run the Deployment GitHub Workflow
```

Comment:
This one is given as an option to do at the end of the script, so maybe we can modify this to indicate as much.